common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -206,6 +206,16 @@ trait CommercialSwitches {
     exposeClientSide = false
   )
 
+  val MembershipEngagementBanner = Switch(
+    SwitchGroup.Commercial,
```

Comment:
I wondered that too, but a) membership doesn't have its own switch group I think and b) this whole thing is a subset of commercial, presumably because it's all about the revenue.